web/wp-content/plugins/tmbi-ad-blocker/is-service-blocked-filter.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * is_service_blocked filter
+ *
+ * Adds various ways to check if a particular ad service should be blocked.
+ *
+ * @package tmbi_ad_blocker
+ */
+
+add_filter( 'is_service_blocked', 'tmbi_ad_blocker_variant_noads', 10, 2 );
+add_filter( 'is_service_blocked', 'tmbi_ad_blocker_blockservice_url_param', 10, 2 );
```

Comment:
@fariasf, Is the same name of this filter intended? As it is calling different methods but with the same parameters.

Reply:
It's on purpose. The filter is the same, but the functions are different. This helps keeping each function small and single-purpose (e.g. `tmbi_ad_blocker_variant_noads` only handles the NOADS variant, while tmbi_ad_blocker_blockservice_url_param`  handles the `blockService` param, without knowing about each other)